src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
-body {
-  margin: 0;
+@import "reset.scss";
+@import "variables.scss";
+@import "functions.scss";
+@import "header.scss";
+@import "logo.scss";
+@import "nav.scss";
+@import "cards.scss";
+@import "card.scss";
+@import "stars.scss";
```

Comment:
Move all imports to style.css and connect style.css to HTML. Why imports are in the file for the BEM block `main`